src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.resx

```diff
@@ -127,13 +127,13 @@
   </data>
   <assembly alias="System.Drawing" name="System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
   <data name="lblLanguageVersion.Location" type="System.Drawing.Point, System.Drawing">
-    <value>6, 23</value>
+    <value>6, 30</value>
```

Comment:
Are the large number of location/size changes here related to DPI settings in the designer? If so, have you validated that the UI still looks good at 100% scaling?

Reply:
Hmm, good question. I'll double-check.

Reply:
I'm going to back out these changes just to be safe.

Reply:
I backed out these changes, restarted VS at 100% scaling, and the designer put them right back. So I'm going to assume these are fine.